crates/build/Cargo.toml

```diff
@@ -12,3 +12,4 @@ tokio = { version = "1.11", features = [ "full" ] }
 spin-loader = { path = "../loader" }
 subprocess = "0.2.8"
 tracing = { version = "0.1", features = [ "log" ] }
+toml = "0.5.9"
```

Comment:
nit: this can probably be removed now?  (This would allow you to revert the Cargo.lock file as well.)